servicetalk-examples/http/traffic-resilience/src/main/java/io/servicetalk/examples/http/traffic/resilience/TrafficResilienceServerTerminalsExample.java

```diff
@@ -36,8 +39,9 @@ public static void main(String[] args) throws Exception {
                         })
                         .build();
 
-        HttpServers.forPort(0)
+        HttpServers.forPort(8080)
                 .appendNonOffloadingServiceFilter(resilienceFilter)
-                .listenAndAwait((ctx, request, responseFactory) -> Single.succeeded(responseFactory.ok()));
+                .listenAndAwait((ctx, request, responseFactory) -> Single.succeeded(responseFactory.ok()))
```

Comment:
Consider using `listenBlockingAndAwait` in all examples, that will allow you to remove `Single.succeeded`